### PR TITLE
Enable css for navigation bar and category buttons

### PR DESCRIPTION
**Pull request by tonycarpenter21**
What (if any) features are you implementing?
- Styled homepage and navigation bar

What (if anything) did you refactor?
- Nothing

Were there any issues that arose?
- No

Is there anything that you need from your teammate?
- No

Any other comments, questions, or concerns?
- No

### CODE REVIEW

File: src/Categories/Categories.scss [hunk 1]
```diff
@@ -1,10 +1,8 @@
...
-  flex-direction: row;
   align-items: center;
-  justify-content: space-between;
+  justify-content:space-around;
   flex-wrap: wrap;
-  height: 65%;
-  width: 66%;
-  margin-top: 2%;
+  text-align: center;
+  padding: 25px;
```
Comment (mr-ryan12) [inline src/Categories/Categories.scss:7]: Thank you for these adjustments!

File: src/CategoryCard/CategoryCard.scss [hunk 2]
```diff
@@ -1,7 +1,15 @@
 .category-card {
```
Comment (StevieAmb) [inline src/CategoryCard/CategoryCard.scss:1]: Awesome use of CSS styling, you are highly skilled!

File: src/CategoryCard/CategoryCard.scss [hunk 3]
```diff
@@ -1,7 +1,15 @@
...
-  width: 19%;
-  border: 5px solid black;
+  width: 30%;
+  padding: 30px;
+  margin: 25px;
+  border-radius: 5px;
+  text-transform: uppercase;
+  font-weight: 900;
+  box-shadow: 5px 5px 0px rgba(0, 0, 0, .25);
+  cursor: pointer;
```
Comment (mr-ryan12) [inline src/CategoryCard/CategoryCard.scss:12]: Really like the way the category buttons are styled!

File: src/CategoryCard/CategoryCard.scss [hunk 4]
```diff
@@ -1,7 +1,15 @@
...
   height: 36%;
-  width: 19%;
-  border: 5px solid black;
+  width: 30%;
+  padding: 30px;
+  margin: 25px;
+  border-radius: 5px;
+  text-transform: uppercase;
+  font-weight: 900;
+  box-shadow: 5px 5px 0px rgba(0, 0, 0, .25);
```
Comment (StevieAmb) [inline src/CategoryCard/CategoryCard.scss:11]: Thank you for teaching us about hex codes and how the numbers are decided and what they mean!

File: src/Navigation/Navigation.scss [hunk 5]
```diff
@@ -2,5 +2,17 @@
...
+}
+
+.logo-accent {
+  color: red;
+}
+
+@media screen and (min-device-width: 0px) and (max-device-width: 800px) {
+  .navigation-bar {
+    justify-content: center;
+  }
```
Comment (mr-ryan12) [inline src/Navigation/Navigation.scss:17]: Nice use of responsive design!

File: src/Navigation/Navigation.scss [hunk 6]
```diff
@@ -2,5 +2,17 @@
...
+  padding: 20px;
+  border-bottom: 5px solid red;
+  display: flex;
+}
+
+.logo-accent {
+  color: red;
+}
+
+@media screen and (min-device-width: 0px) and (max-device-width: 800px) {
```
Comment (StevieAmb) [inline src/Navigation/Navigation.scss:14]: Great use of media queries and responsive design.

File: src/Navigation/Navigation.tsx [hunk 7]
```diff
@@ -4,9 +4,18 @@ import './Navigation.scss'
 const Navigation = () => {
   return (
     <nav className="navigation-bar">
-      some stuff 
+      <h1 className="logo">
```
Comment (StevieAmb) [inline src/Navigation/Navigation.tsx:7]: Nice work of making the title display block!

**Review by StevieAmb — APPROVED** [review]

**Comment by mr-ryan12** [comment]
Thank you for making these changes. Next steps include fetching data from the API!

**Pull request merged**